### PR TITLE
Add tileset doc for Redshift

### PR DESCRIPTION
**Pull request by alex-graciano**
Added overview documentation for the tiler in Redshift. Added point aggregation tileset so far.

### CODE REVIEW

File: app/content/analytics-toolbox-redshift/sql-reference/tiler.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Additionally, there is a row identified with `Z=-1` which contains metadata about the tileset in the `DATA` column in JSON format. It contains the following properties:
+
+* `bounds`: geographical extents of the source as a string in `Xmin, Ymin, Xmax, Ymax` format.
+* `center`: center of the geographical extents as `X, Y, Z`, where the `Z` represents the zoom level where a single tile spans the whole extents size.
+* `zmin`: minimum zoom level in the tileset.
+* `zmax`: maximum zoom level in the tileset.
+* `tilestats`: stats about the feature's properties. In addition to its name (`attribute`) and `type`, it contains `min`, `max`, `average` and `sum`.
+
+{{% customSelector %}}
```
Comment (Jesus89) [inline app/content/analytics-toolbox-redshift/sql-reference/tiler.md:45]: Remove `{{% customSelector %}}`

**Review by Jesus89 — APPROVED** [review]
LGTM

Added just a comment. It should be fixed in the toolbox for future PRs in the tiler redshift.

File: app/content/analytics-toolbox-redshift/sql-reference/tiler.md (hunk 1 above)
Comment (alex-graciano) [reply app/content/analytics-toolbox-redshift/sql-reference/tiler.md:45]: I noticed that in other modules like `s2`. Should they be removed as well?

Comment (Jesus89) [reply app/content/analytics-toolbox-redshift/sql-reference/tiler.md:45]: Yes, for snowflake and redshift, please

**Comment by Jesus89** [comment]
:+1:

**Pull request merged**